main.c

```diff
@@ -85,12 +67,14 @@ static void seat_update_selection(struct slurp_seat *seat) {
 }
 
 static void seat_set_outputs_dirty(struct slurp_seat *seat) {
+	struct slurp_state *state = seat->state;
 	struct slurp_output *output;
 	wl_list_for_each(output, &seat->state->outputs, link) {
 		if (box_intersect(&output->logical_geometry,
 			&seat->pointer_selection.selection) ||
 		    box_intersect(&output->logical_geometry,
-			&seat->touch_selection.selection)) {
+			&seat->touch_selection.selection) ||
+			state->crosshairs) {
```

Comment:
this should probably check that the selection is on the output, but I haven't gotten that to work yet.